htdocs/commande/list.php

```diff
@@ -449,7 +449,7 @@
 if (($search_categ_cus > 0) || ($search_categ_cus == -2)) {
 	$sql .= ' LEFT JOIN '.MAIN_DB_PREFIX."categorie_societe as cc ON s.rowid = cc.fk_soc"; // We'll need this table joined to the select in order to filter by categ
 }
-$sql .= ', '.MAIN_DB_PREFIX.'commande as c';
+$sql .= 'RIGHT JOIN '.MAIN_DB_PREFIX.'commande as c ON c.fk_soc = s.rowid';
```

Comment:
We must avoid right join that generated a very difficult way to understand readability of requests. Only inner and left join should be used.

Reply:
OK, I guess I will simply replace every comma-join by an explicit INNER JOIN.

Reply:
> OK, I guess I will simply replace every comma-join by an explicit INNER JOIN.

Yes,for similar result, comma-join can be replaced with inner join. It is the other way to write inner join.